keyboards/kinesis/kint2pp/config.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "../config.h"
```

Comment:
This line _shouldn't_ be needed

```suggestion

#include "../config.h"
```

Reply:
Done